tools/react-integration-tester/src/__tests__/cli.e2e.test.ts

```diff
@@ -308,19 +319,53 @@ describe('rit CLI e2e', () => {
 
         return tsPathAliases;
       }
+
+      /**
+       * @param {import('@jest/types').Config.InitialOptions} config - project origin provided config
+       * @param {import('@jest/types').Config.InitialOptions} workspaceConfig - workspace preset config
+       */
+      function createOriginProjectConfig(config, workspaceConfig) {
+        // no preset property, we will use configuration as is
+        if (!config.preset) {
```

Comment:
not every project has `preset` set, this creates resolved jest config based on origin setup to make it repo agnostic